circle.yml

```diff
@@ -23,6 +23,8 @@ test:
   pre:
 
   override:
+    - echo decrypting config
+    - cd wp-e2e-tests && npm run decryptconfig
     - echo running test command "./run.sh -R -C $RUN_ARGS"
     - cd wp-e2e-tests && git checkout origin/${E2E_BRANCH-master} && ./run.sh -R -C $RUN_ARGS
```

Comment:
I think we should run `decryptconfig` after checkout. For testing you can try set `NODE_CONFIG={}` when you run the tests: `&& NODE_CONFIG={} ./run.sh -R -C $RUN_ARGS`

Reply:
Thanks, fixed